client/login/wp-login/login-links.jsx

```diff
@@ -137,7 +137,7 @@ export class LoginLinks extends React.Component {
 				icon={ true }
 				onClick={ this.recordHelpLinkClick }
 				target="_blank"
-				href="https://en.support.wordpress.com/security/two-step-authentication/"
+				href="https://wordpress.com/support/security/two-step-authentication/"
```

Comment:
Does this need `localizeUrl()` (I'm not sure if there's cases that do or don't need it)?

Reply:
No because it uses `<ExternalLink>`